### DISCUSSION TITLE
[Configurations][Kustomize] Error when deploying after conversion to kustomize

### DISCUSSION THREAD

**Discussion by frengooo** [mia-platform/community#229] [Q&A] [answered]
### Which section?

Console

### Which is the version?

v10.7.1

### Description

Hi,
I converted a project to `kustomize` following [the official documentation](https://docs.mia-platform.eu/docs/development_suite/set-up-infrastructure/kustomize/migrate#docusaurus_skipToContent_fallback).

I followed the steps but the first deploy failed during the execution of the command:
```bash
mlp kustomize "${OVERLAY_PATH}" -o "${DESTINATION_PATH}/kustomize-output.yaml"
```

The error message is the following one:
```
accumulating resources: accumulation err='merging resources from 'nexus-gcloud.secret.yaml': may not add resource with an already registered id: Secret.v1.[noGrp]/nexus-gcloud.[noNs]': '/builds/clients/{{CLIENT_NAME}}/configuration/overlays/development/nexus-gcloud.secret.yaml' must be a directory so that it can used as a build root
```

Can you please take a look?

Thanks.

**Comment by davidebianchi** [comment, marked as answer]
Hi @silversoul93! Have you two secret with the same name? Can you try to remove one of the two, and retry the deploy?

**Reply by frengooo** [reply]
I deleted the `configuration/nexus-gcloud.secret.yaml` file and it works.

Thanks 💪
